Library/Homebrew/extend/os/mac/hardware/cpu.rb

```diff
@@ -114,7 +114,7 @@ def arm_family
         end
       end
 
-      def intel_family
+      def intel_family(_family = nil, _cpu_model = nil)
```

Comment:
This needs to match the arity of the same method in Library/Homebrew/extend/os/linux/hardware/cpu.rb